nx.json

```diff
@@ -85,5 +86,35 @@
         "linter": "eslint"
       }
     }
+  },
+  "release": {
+    "projects": [
+      "packages/*",
+      "!packages/plugin-lighthouse",
+      "!packages/nx-plugin"
+    ],
+    "projectsRelationship": "fixed",
+    "changelog": {
+      "automaticFromRef": true,
+      "projectChangelogs": false,
+      "workspaceChangelog": {
+        "createRelease": "github"
+      }
+    },
+    "git": {
+      "commit": true,
+      "push": true,
+      "pushRemote": "origin",
+      "tag": true,
+      "commitMessage": "release: {version} [skip ci]",
+      "commitArgs": "--no-verify"
```

Comment:
Do we need the `--no-verify` flag? Our commit hooks are made up of Prettier and Commitlint (the `release: ...` message should be compatible), so I guess verification should pass for these automated commits.

Reply:
Not really if you support the `release` keyword.
Just if you want to save few seconds :)

Reply:
In that case I would remove `--no-verify`.